src/agent/mock_agent.py

```diff
@@ -89,13 +91,18 @@ def get_mock_stats_from_user(self):
     def get_containers_summary(self) -> List[ContainerSummary]:
         summaries = []
         for container_info in self.containers_info:
+            mem = int(container_info["memory_usage"])
+            cpu_percent = container_info["cpu_usage"]
+
             container_summary = ContainerSummary(
                 id=container_info["id"],
                 name=container_info["name"],
                 image=container_info["image"],
                 status=container_info["status"],
```

Comment:
@Ilargi12 what are all possible values of `status`?
- [States](https://stackoverflow.com/a/32428199)
- [Statuses](https://stackoverflow.com/a/34474518)

Reply:
All possible states are mentioned in: [States](https://stackoverflow.com/questions/32427684/what-are-the-possible-states-for-a-docker-container/32428199#32428199).

If you want we can discuss verifying each string for status